packages/opencensus-core/test/test-stats.ts

```diff
@@ -225,5 +226,40 @@ describe('Stats', () => {
          globalStats.record(measurments, tagMap);
          assert.equal(testExporter.recordedMeasurements.length, 0);
        });
+
+    it('should record against implicit context when set', () => {
+      const tags = new TagMap();
+      tags.set(tagKeys[0], {value: 'value1'});
+      tags.set(tagKeys[1], {value: 'value2'});
+      const measurement = {measure, value: 1};
+      tagger.withTagContext(tags, () => {
+        globalStats.record([measurement]);
+      });
+
+      assert.strictEqual(testExporter.recordedMeasurements.length, 1);
+      assert.deepEqual(testExporter.recordedMeasurements[0], measurement);
+      aggregationData =
+          testExporter.registeredViews[0].getSnapshot(
+              [{value: 'value1'}, {value: 'value2'}]) as LastValueData;
```

Comment:
What's the role of the `as LastValueData`, since it looks like `aggregationData` already has that type? Can that safely be removed here and below?

Reply:
The return type of ```getSnapshot()``` method is ```AggregationData```, which is combination of four types ```SumData|CountData|LastValueData|DistributionData```. The conversion (```as LastValueData```) is added to get exact type of ```AggregationData```.

Reply:
Make sense, thanks!